src/scene/scene_phongmaterial.js

```diff
@@ -210,6 +210,7 @@ pc.extend(pc, function () {
             this.specularMapTint = false;
             this.emissiveMapTint = false;
             this.emissiveIntensity = 1;
+            this.mask = 1;

```

Comment:
Should this be in pc.Material? Shouldn't all materials be able to specify which lights they are lit by? Not just Phong Materials?